clap-utils/src/keypair.rs

```diff
@@ -159,11 +167,29 @@ pub fn presigner_from_pubkey_sigs(
     })
 }
 
+pub fn signer_from_path_allow_null(
```

Comment:
Does this make sense as an interface or should we just cut to the chase and do a `_with_config` ?

Reply:
While I don't have other config options particularly in mind, I like the `_with_config` method name better. In usage, `signer_from_path_allow_null` sort of reads like the path is allowed to be null as opposed to the signer (and NullSigner may not itself appear anywhere in the downstream).
I think `Config { allow_null_signer: bool}` works a bit better